t3f/ops.py

```diff
@@ -1132,3 +1136,50 @@ def gather_nd(tt, indices):
     tt_elements = tf.matmul(tt_elements, core_slices)
   tt_elements = tf.reshape(tt_elements, tf.shape(indices)[:-1])
   return tt_elements
+
+def renormalize_cores(tt):
+    """Renormalizes tt cores to make them of the same Frobenius norm.
+
+    Does not change tt, but makes computations with cores more stable.
```

Comment:
Doesn't change the tensor represented by the `tt` object, but renormalizes the TT-cores to make further computations more numerically stable.